### PR TITLE
Adding DRAFT status to DC files

### PR DESCRIPTION
**Pull request by fsundermeyer**

### CODE REVIEW

File: xml/concept-klp.xml [hunk 1]
```diff
@@ -20,8 +20,8 @@
    <dm:bugtracker>
     <dm:url>https://bugzilla.suse.com/enter_bug.cgi</dm:url>
     <dm:component>Documentation</dm:component>
-    <dm:product>Product Name</dm:product>
-    <dm:assignee>assignee@suse.com</dm:assignee>
+    <dm:product>PUBLIC SUSE Linux Enterprise Server 15 SP3</dm:product>
+    <dm:assignee>dpopov@suse.com</dm:assignee>
```
Comment (taroth21) [inline xml/concept-klp.xml:24]:
```suggestion
    <dm:assignee>dmitri.popov@suse.com</dm:assignee>
```

File: xml/concept-zypper.xml [hunk 2]
```diff
@@ -20,8 +20,8 @@
    <dm:bugtracker>
     <dm:url>https://bugzilla.suse.com/enter_bug.cgi</dm:url>
     <dm:component>Documentation</dm:component>
-    <dm:product>Product Name</dm:product>
-    <dm:assignee>assignee@suse.com</dm:assignee>
+    <dm:product>PUBLIC SUSE Linux Enterprise Server 15 SP3</dm:product>
+    <dm:assignee>dpopov@suse.com</dm:assignee>
```
Comment (taroth21) [inline xml/concept-zypper.xml:24]:
```suggestion
    <dm:assignee>dmitri.popov@suse.com</dm:assignee>
```

File: xml/reference-zypper-install.xml [hunk 3]
```diff
@@ -20,8 +20,8 @@
    <dm:bugtracker>
     <dm:url>https://bugzilla.suse.com/enter_bug.cgi</dm:url>
     <dm:component>Documentation</dm:component>
-    <dm:product>Product Name</dm:product>
-    <dm:assignee>assignee@suse.com</dm:assignee>
+    <dm:product>PUBLIC SUSE Linux Enterprise Server 15 SP3</dm:product>
+    <dm:assignee>dpopov@suse.com</dm:assignee>
```
Comment (taroth21) [inline xml/reference-zypper-install.xml:24]:
```suggestion
    <dm:assignee>dmitri.popov@suse.com</dm:assignee>
```

File: xml/task-check-cpu-temperature.xml [hunk 4]
```diff
@@ -24,8 +24,8 @@
     <dm:bugtracker>
      <dm:url>https://bugzilla.suse.com/enter_bug.cgi</dm:url>
      <dm:component>Documentation</dm:component>
-     <dm:product>Product Name</dm:product>
-     <dm:assignee>assignee@suse.com</dm:assignee>
+     <dm:product>PUBLIC SUSE Linux Enterprise Server 15 SP3</dm:product>
+     <dm:assignee>dpopov@suse.com</dm:assignee>
```
Comment (taroth21) [inline xml/task-check-cpu-temperature.xml:28]:
```suggestion
     <dm:assignee>dmitri.popov@suse.com</dm:assignee>
```

File: xml/task-create-systemd-timers.xml [hunk 5]
```diff
@@ -19,7 +19,7 @@
    <dm:bugtracker>
     <dm:url>https://bugzilla.suse.com/enter_bug.cgi</dm:url>
     <dm:component>Documentation</dm:component>
-    <dm:product>Product Name</dm:product>
+    <dm:product>PUBLIC SUSE Linux Enterprise Server 15 SP3</dm:product>
     <dm:assignee>dpopov@suse.com</dm:assignee>
```
Comment (taroth21) [inline xml/task-create-systemd-timers.xml:23]:
```suggestion
    <dm:assignee>dmitri.popov@suse.com</dm:assignee>
```

**Review by taroth21 — CHANGES_REQUESTED** [review]
IIRC, the e-mail-address for bugs needs to be the one registered/used within Bugzilla, therefore changing the one for Dima.

**Comment by sknorr** [comment]
In a way, this is fun -- I specifically updated Docserv2 to not add `--draft` to the command line of docs marked as `beta` [just days earlier](https://github.com/openSUSE/docserv/commit/24382ccb285857f5e653b56f688a5f2423a0e319), because `--draft` also triggers `--meta` + `--remarks` as well, and both of those imo look a bit unprofessional on a published website.

So,

1. Sorry for creating this sort of busywork for you.
2. Should we do something about the fact that DAPS includes meta information and remarks in every `--draft` build? Or do think that is still expected?

**Pull request merged**